04_environment/index.tex

```diff
@@ -1 +1,283 @@
+    \item Instead of a specified return, probabilistic return has been implemented which adds a randomness element to the game to prevent pattern recognition. These probabilities can be tied to the population, disasters, forecasting and number of animals hunted.  
+\end{enumerate}
+
+\subsubsection{Tier System}
+
+The deer hunting and fishing is divided into tiers (the number of fish that can be caught in a given day) which depends on the total input resources. Where the “zero” tier represents the cost of getting to the hunting location.
+
+Beyond the zero tier, the cost of catching another deer/fish will decrease as it is easier to catch the second animal than the first since the agents have already arrived at the hunting location. It should be noted there is a limit to how many animals the agents can catch in a given day.
+
+The focus of fishing is to avoid risk at the expense of payoff. This means that all tier requirements are lower than the deers. However, the return is also lower and in addition, the “zero” tier is also removed. Thus, if only 1 island goes fishing they are still expected to have a low return as long as they have enough resources to catch the first fish.
+
+The benefit of deer hunting is higher payoffs. However, these improved returns come at the cost of significantly higher tier thresholds and a “zero” tier. Therefore, it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.
+
+\begin{equation}
+Increments\ of\ catching\ n\ deer's=
+\left( \begin{array}{ll@{}}
+n=0 -> \Delta^0 = 1 \\
+n=1 -> \Delta^1 = 0.8^1 \\
+n=2 -> \Delta^2 = 0.8^2 = 0.64 \\
+n=2 -> \Delta^3 = 0.8^3 = 0.512 \\
+n=4 -> \Delta^4 = 0.8^4 = 0.4096\\
+\end{array} \right) 
+\label{eq:Cumulative Cost 1}
+\end{equation}
+
+\begin{equation}
+Cumulative\ deer \ cost \sum_{n=0}^{n} \Delta^{n} = \Delta^{0} + \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + ....
+\label{eq:Cumulative Cost 2}
+\end{equation}
+
+\begin{equation}
+Cumulative\ fishing\ cost \ = \ \sum_{n=1}^{n} \Delta^{n} = \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + \Delta^{5} + ....
+\label{eq:Cumulative Cost 3}
+\end{equation}
+
+Equations that show the formulation of the decay function for tier system.
+
+The tier system is shown in Equation~\eqref{eq:Cumulative Cost 1}, Equation~\eqref{eq:Cumulative Cost 2} and Equation~\eqref{eq:Cumulative Cost 3}, where $n$ stands for the tiers, with decay costs ($\Delta$) of $0.8$ for both the deer hunt and fishing. Fishing cost starts at $n$ is equal to $1$ due to the reduced cost.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.9\textwidth]{04_environment/Images/Foraging Tier System.PNG}
+    \caption{Foraging Tier System}
+    \label{fig:Foraging Tier System}
+\end{figure}
+
+Higher tiers result in a lower cost per animal, however, the total cost increases up to a daily catch limit.
```

Comment:
".. However,"

it is advised that you start a _new_ sentence whenever you use "however"